pkg/reconciler/apis/apibinding/apibinding_reconcile_test.go

```diff
@@ -956,7 +956,8 @@ func (b *bindingBuilder) Build() *apisv1alpha1.APIBinding {
 
 func (b *bindingBuilder) WithClusterName(clusterName string) *bindingBuilder {
 	//TODO (shawn-hurley): We may need to change how we set this
-	b.ClusterName = clusterName
+	// nolint:staticcheck
```

Comment:
side-comment: it looks like this check fails only when updating a field that's deprecated, not when creating a struct literal and setting it ...